### PR TITLE
[DOC Canary] Added comments for ember-testing helper scripts

### PR DESCRIPTION
**Pull request by pranjalk**
Added comments helper scripts for issue https://github.com/emberjs/ember.js/issues/13862

### CODE REVIEW

File: packages/ember-testing/lib/helpers/current_route_name.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/*
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/current_route_name.js]: Can you move this down to just above `currentRouteName` function below?

Also, the comment needs to start with `/**` for YUIDoc to properly parse it as documentation.

File: packages/ember-testing/lib/helpers/pause_test.js [hunk 2]
```diff
@@ -1,3 +1,17 @@
+/*
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/pause_test.js]: Can you move this down to just above the `pauseTest` function below, and update the comment open to be `/**`?

File: packages/ember-testing/lib/helpers/trigger_event.js [hunk 3]
```diff
@@ -1,3 +1,23 @@
+/*
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/trigger_event.js]: Same as above

**Comment by rwjblue** [comment]
Can you:

* add the following snippet to the top of each helper file:

```js
/**
@module ember
@submodule ember-testing
*/
```

* Remove the existing documentation in https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/lib/helpers.js that this is replacing.

Thank you!

**Comment by rwjblue** [comment]
Looks great! Would you mind squashing the commits?

**Comment by pranjalk** [comment]
HI! it seems that I am having trouble with squashing the commits

**Comment by pranjalk** [comment]
I think its done??

**Comment by rwjblue** [comment]
Awesome, thank you very much!

**Pull request merged**